server/src/main/java/org/hyperskill/community/flashcards/card/response/PermittedAction.java

```diff
@@ -0,0 +1,4 @@
+package org.hyperskill.community.flashcards.card.response;
+
+public record PermittedAction(ActionType action, String uri) {
```

Comment:
what is the URI for?

Reply:
as discussed for frontend calls to REST-endpoints - let's see, if we need it - but for now ok.

Reply:
poor man's HATEOAS :smile: We can remove it.